descope/client/config.go

```diff
@@ -61,15 +64,18 @@ type Config struct {
 	SessionCookieName string
 }
 
-func (c *Config) setProjectID() string {
+func (c *Config) setProjectID() (projectID string, ok bool) {
 	if c.ProjectID == "" {
-		if projectID := utils.GetProjectIDEnvVariable(); projectID != "" {
-			c.ProjectID = projectID
-		} else {
-			return ""
-		}
+		c.ProjectID = utils.GetProjectIDEnvVariable()
+	}
+	c.ProjectID = strings.TrimSpace(c.ProjectID)
+	if c.ProjectID == "" {
+		return "", false
+	}
+	if c.ProjectID == "-" {
```

Comment:
I dont like this hack
Lets add another prop, to specify that we are deliberatly ignoring project id